functions/sum-of-inputs.py

```diff
@@ -0,0 +1,14 @@
+def sum_of_inputs(x,y):                 # To calculate the sume of given value.	
```

Comment:
Use function doc string based on this reference https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html